cogs/logs.py

```diff
@@ -0,0 +1,106 @@
+import nextcord
+from nextcord.ext import commands
+import aiosqlite
+import asyncio
+from datetime import datetime
+
+class Logs(commands.Cog):
+    def __init__(self, client):
+        self.client = client
+
+    @commands.Cog.listener()
+    async def on_ready(self):
+        print("Online!")
+        setattr(self.client, "db", await aiosqlite.connect("main.db"))
+        await asyncio.sleep(1)
+        async with self.client.db.cursor() as cursor:
+            await cursor.execute("CREATE TABLE IF NOT EXISTS logging (channel INTEGER, guild INTEGER)")
+        await self.client.db.commit()
+
+    @commands.Cog.listener()
+    async def on_message_delete(self, message):
+        guild = message.guild
+        async with self.client.db.cursor() as cursor:
+            await cursor.execute("SELECT channel FROM logging WHERE guild = ?", (guild.id,))
+            channel = await cursor.fetchone()
+            msgDelete = self.client.get_channel(channel[0])
+            embed = nextcord.Embed(title = f"{message.author}'s Message was Deleted", description = f"Deleted Message: {message.content}\nAuthor: {message.author.mention}", timestamp = datetime.now(), color = nextcord.Colour.red())
+            embed.set_author(name = message.author.name, icon_url = message.author.display_avatar)
+            embed.set_footer(text=str(message.author.id))
+            await msgDelete.send(embed=embed)
+
+    @commands.Cog.listener()
+    async def on_message_edit(self, before, after):
```

Comment:
quick thing, there is a possibility of `before` being None on any update or edit event.